### PR TITLE
Time point list: Added a Site column to the table; Redmine 10938

### PR DESCRIPTION
**Pull request by MounaSafiHarab**
Added a Site column to the timepoint list; this will become useful if some timepoints/visits took place at other sites...

### CODE REVIEW

File: modules/timepoint_list/php/NDB_Menu_timepoint_list.class.inc [hunk 1]
```diff
@@ -77,6 +78,12 @@ class NDB_Menu_timepoint_list extends NDB_Menu
                 $this->tpl_data['timePoints'][$x]['currentStatus'] = $timePoint->getCurrentStatus();
                 $this->tpl_data['timePoints'][$x]['currentDate'] = $timePoint->getCurrentDate();
                 
+
+                // get Center Name
+		$centerID = $timePoint->getData('CenterID');
+		$center = $DB->pselectRow("SELECT CenterID as ID, PSCArea as Name FROM psc WHERE CenterID =:cid", array('cid' => $centerID));
```
Comment (MounaSafiHarab) [inline modules/timepoint_list/php/NDB_Menu_timepoint_list.class.inc]: @driusan 
Is the CenterName the Name column or the PSCArea column in the psc table?

Comment (driusan) [inline modules/timepoint_list/php/NDB_Menu_timepoint_list.class.inc]: I think "Alias" (the 3 digit short code)

**Comment by driusan** [comment]
(It was @LeighMac's suggestion, not mine.)

File: modules/timepoint_list/php/NDB_Menu_timepoint_list.class.inc [hunk 2]
```diff
@@ -77,6 +78,12 @@ class NDB_Menu_timepoint_list extends NDB_Menu
                 $this->tpl_data['timePoints'][$x]['currentStatus'] = $timePoint->getCurrentStatus();
                 $this->tpl_data['timePoints'][$x]['currentDate'] = $timePoint->getCurrentDate();
                 
+
+                // get Center Name
+		$centerID = $timePoint->getData('CenterID');
+		$center = $DB->pselectRow("SELECT CenterID as ID, Alias FROM psc WHERE CenterID =:cid", array('cid' => $centerID));
```
Comment (driusan) [inline modules/timepoint_list/php/NDB_Menu_timepoint_list.class.inc]: Can't these two lines just be replaced by $timePoint->getPSC() ?

Comment (MounaSafiHarab) [inline modules/timepoint_list/php/NDB_Menu_timepoint_list.class.inc]: wouldn't this return the Site name, and I thought we agreed we should display the 3 letter alias instead?

Comment (driusan) [inline modules/timepoint_list/php/NDB_Menu_timepoint_list.class.inc]: You're right, my bad!

We have too many ways to refer to sites, I'm going to add something to the meeting agenda about it..

**Pull request merged**